sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/models/CommunicationUserIdentifierWithTokenResult.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.identity.models;
+
+import com.azure.communication.common.CommunicationUserIdentifier;
+import com.azure.core.credential.AccessToken;
+
+/** The CommunicationUserIdentifierWithAccessTokenResult model. */
+public final class CommunicationUserIdentifierWithTokenResult {
```

Comment:
C# managed to keep this class as package private. Can we do the same? Try not to expose this class if possible otherwise a better name is required.

Reply:
This is a manually created class that allows us to return both the user identifier and token in a single value. C# supports tupples while Java does not, so we had to encapsulate the values into a class.